src/toolkits/object_detection/object_detector.cpp

```diff
@@ -659,8 +659,11 @@ variant_type object_detector::predict(
   // Convert data to SFrame
   std::string image_column_name = read_state<flex_string>("feature");
   gl_sframe sframe_data = convert_types_to_sframe(data, image_column_name);
+  gl_sframe sframe_image_data(
```

Comment:
please comment on why you need to extract the sframe. Otherwise, without the context of #2830, other ppl may feel strange.

Reply:
good!